src/rust/engine/src/nodes.rs

```diff
@@ -496,11 +497,20 @@ impl ExecuteProcess {
       .map(|path: String| PathBuf::from(path))
       .collect();
 
+    let timeout_str = externs::project_str(&value, "timeout");
```

Comment:
It's slightly sad that this goes through a float -> string -> float round-trip, but I'm pretty sure I don't care enough to do the whole round-tripping floats across the ffi boundary piece :)